tests/transpiler/qiskit/test_conversions_qiskit.py

```diff
@@ -231,8 +243,24 @@ def test_qiskit_roundtrip_noncontig():
 
 def test_100_random_qiskit():
     """Test converting 100 random qiskit circuits to cirq."""
+
+    gates_to_skip = ["c3sqrtx"]
+
+    def _circuit_contains_invalid_gate(qiskit_circuit):
+        for gate in gates_to_skip:
+            if len(qiskit_circuit.get_instructions(gate)) > 0:
+                return True
+        return False
+
+    def _generate_valid_qiskit_circuit():
+        qiskit_circuit = random_circuit(4, 1, max_operands=2)
```

Comment:
Can you double check there aren't any 5 qubit gates that we would be leaving out?

Reply:
Increased qubits to 5 and ops to 4, works still